WORKING_GROUPS.md

```diff
@@ -0,0 +1,137 @@
+
+The intention of the agenda is not to approve or review all patches, that should happen continuously on GitHub and be handled by the larger group of Collaborators.
+
+Any community member or contributor can ask that something be added to the next meeting's agenda by logging a GitHub Issue. Any Collaborator, WG member, or moderator can add the item to the agenda by adding the ***WG-agenda*** tag to the issue.
+
+Prior to each WG meeting, the moderator will share the Agenda with members of the WG. WG members can add any items they like to the agenda at the beginning of each meeting. The moderator and the WG cannot veto or remove items.
+
+The WG may invite persons or representatives from related projects to participate in a non-voting capacity.
+
+The moderator is responsible for summarizing the discussion of each agenda item, and commits it as a pull request after the meeting.
+
+### Consensus Seeking Process
+
+The Working Group follows a [Consensus Seeking][] decision-making model.
+
+When an agenda item has appeared to reach a consensus, the moderator will ask "Does anyone object?" as a final call for dissent from the group consensus.
+
+If an agenda item cannot reach a consensus: a WG member can call for either a closing vote, or a vote to table the issue until the next meeting. The call for a vote must be seconded by a majority of the WG, or else the discussion will continue. A simple majority wins.
+
+Note that changes to WG membership require unanimous consensus. See the "Working Group Membership" section above for more information.
+
+<a id="developers-certificate-of-origin"></a>
+## Developer's Certificate of Origin 1.1
+
+*Note*: The DCO is mandatory for all OpenJS Foundation projects.
+
+By making a contribution to this project, I certify that:
+
+* (a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or
+
+* (b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or
+
+* (c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.
+
+* (d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.
+
+### Moderation Policy
+
+The [OpenJS Moderation Policy] applies to this Working Group.
```

Comment:
👉 We have yet to draft the moderation policy and link it here.

Reply:
Other than this, I'll give this my approval – and would appreciate a review of my commits! 👌